src/App.css

```
@@ -2,7 +2,7 @@
 
 @font-face {
   font-family: Vazir;
-  src: url("./fonts/Vazir/Vazir-Regular.ttf");
+  src: url("fonts/Vazir/Vazir-Regular.ttf");
```

Reverting this change.
```suggestion
  src: url("./fonts/Vazir/Vazir-Regular.ttf");
```